dirsrvtests/tests/suites/subentries/subentries_test.py

```diff
@@ -0,0 +1,201 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 RED Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ----
+
+# Author: Anton Bobrov <abobrov@redhat.com>
+
+import logging
+import pytest
+import os
+import ldap
+from ldap.controls import LDAPControl
+from lib389.utils import *
+from lib389._constants import *
+from lib389.topologies import topology_st as topo
+from lib389.idm.user import UserAccount, UserAccounts
+
+log = logging.getLogger(__name__)
+
+pytestmark = pytest.mark.tier0
+
+"""
+This BooleanControl class is taken from python-ldap,
```

Comment:
I think it would be better to put it in https://github.com/389ds/389-ds-base/blob/master/src/lib389/lib389/_controls.py in case we'll want to use it for lib389/ds* tools, until the fixed version of python-ldap appears in distros.

Reply:
I sent a pull request to python-ldap, see https://github.com/python-ldap/python-ldap/pull/441 so hopefully this wont be necessary.